task2.py

```diff
@@ -0,0 +1,10 @@
+# TODO Найдите количество книг, которое можно разместить на дискете
+result = 1.44
+Kb = result*1024
+b = Kb*1024
```

Comment:
Повторите правила именования переменных, все переменные именуются с маленькой буквы, а заглавными или с заглавной константы. 
Также можно избежать использование лишних переменных и операций, выполнив перемножение в единой строке.
```suggestion
bytes = result * 1024 ** 2
```